dandi/files/bases.py

```diff
@@ -469,6 +480,31 @@ def get_validation_errors(
         else:
             # make sure that we have some basic metadata fields we require
             try:
+                # Ensure latest version of NWB Inspector is installed and used client-side
+                got_inspector_version = True
+                try:
+                    current_version, max_version = self._get_inspector_versions()
+                except Exception as e:  # In case of no internet connection or other error
+                    got_inspector_version = False
+                    lgr.warning(
+                        "Failed to retrieve NWB Inspector version.",
+                        self.filepath,
+                        type(e).__name__,
+                        str(e),
+                    )
+                if got_inspector_version and current_version < max_version:
+                    errors.extend(
```

Comment:
You're only adding one item, so use `errors.append()` and get rid of the brackets around the item.